adapter/pubsub/constants.h

```diff
@@ -0,0 +1,61 @@
+/* * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * *
```

Comment:
I don't think you need this file. It is already present in `adapter/constants.h`.